x/staking/keeper/period_delegation.go

```diff
@@ -37,8 +43,8 @@ func (k Keeper) ValidateNewPeriodDelegation(
 	return periodDelegation, nil
 }
 
-// GetAllPeriodDelegation returns all period delegation by delAddr and valAddr.
-func (k Keeper) GetAllPeriodDelegation(ctx context.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) ([]types.PeriodDelegation, error) {
+// GetAllPeriodDelegationsByDelAndValAddr returns all period delegations by delAddr and valAddr.
+func (k Keeper) GetAllPeriodDelegationsByDelAndValAddr(ctx context.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) ([]types.PeriodDelegation, error) {
```

Comment:
@LuttyYang @jhd2best 
In this PR, I changed the name of function since I need to make a func to get all period delegations regardless of delegator or validator address.